src/plugins/discover/public/application/components/context_app/context_app_legacy.test.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { ContextAppLegacy } from './context_app_legacy';
+import { IIndexPattern } from '../../../../../data/common/index_patterns';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { DocTableLegacy } from '../../angular/doc_table/create_doc_table_react';
+
+describe('ContextAppLegacy test', () => {
+  const hit = {
+    _id: '123',
+    _index: 'test_index',
+    _score: null,
+    _version: 1,
+    _source: {
+      category: ["Men's Clothing"],
+      currency: 'EUR',
```

Comment:
Nit:  Walker Text Ranger's first name ist Cordell and he would never pay in EUR !